src/argilla/server/apis/v1/handlers/datasets.py

```diff
@@ -779,10 +779,11 @@ async def search_dataset_records(
     include: Optional[RecordIncludeParam] = Depends(parse_record_include_param),
     response_statuses: List[ResponseStatusFilter] = Query([], alias="response_status"),
     offset: int = Query(0, ge=0),
-    limit: int = Query(default=LIST_DATASET_RECORDS_LIMIT_DEFAULT, lte=LIST_DATASET_RECORDS_LIMIT_LTE),
+    limit: int = Query(default=LIST_DATASET_RECORDS_LIMIT_DEFAULT, ge=1, le=LIST_DATASET_RECORDS_LIMIT_LE),
```

Comment:
We should add a breaking change tag using old clients (which send limit=0)

Reply:
With this do you mean a new line on  `CHANGELOG.md`?